### PR TITLE
addressed customer questions/comments

### PR DESCRIPTION
**Pull request by mairaw**
addressing LiveFyre comments/questions for the following two related WCF topics:
1 - https://docs.microsoft.com/en-us/dotnet/framework/wcf/find-private-key-tool-findprivatekey-exe
- see... what??? It's blank!

2 - https://docs.microsoft.com/en-us/dotnet/framework/wcf/samples/findprivatekey
- Where is the download link for the "sample code"?
- Repeating Meng_Wang's question - where is the sample code?
- For those of you wandering... you can download the sample code here: https://www.microsoft.com/en-us/download/details.aspx?id=21459

For the service account names, I've changed the references to Windows XP and Windows Server 2003 to generic names. @Rick-Anderson @tdykstra do you know if they are still correct? See the part about ASPNET and NETWORK SERVICE accounts.

Please use rich diff to see the changes.

### CODE REVIEW

**Comment by tdykstra** [comment]
Rick may know but is on vacation.  If you don't get an answer from him, I'd ask [@]shirhatti

File: docs/framework/wcf/samples/findprivatekey.md [hunk 1]
```diff
@@ -20,85 +12,93 @@ author: "Erikre"
...
- This example finds the filename of the certificate with a thumbprint of "03 33 98 63 d0 47 e7 48 71 33 62 64 76 5c 4c 9d 42 1d 6b 52", in the Personal store of the Local Computer.  
-  
-```  
-FindPrivateKey My LocalMachine -t "03 33 98 63 d0 47 e7 48 71 33 62 64 76 5c 4c 9d 42 1d 6b 52" –c  
-```  
-  
-## See Also
+> FindPrivateKey is a sample that needs to be compiled prior to use. See the [To build the FindPrivateKey project](#to-build-the-findprivatekey-project) section for instructions on how to build the FindPrivateKey tool.
+
+X.509 certificates are installed by an Administrator or any user in the machine. However, the certificate may be accessed by a service running under a different account. For example, the ASPNET on Windows clients or the NETWORK SERVICE accounts on Windows Server machines.
```
Comment (tdykstra) [inline docs/framework/wcf/samples/findprivatekey.md]: Missing "account" after ASPNET

**Comment by zhenlan** [comment]
> For the service account names, I've changed the references to Windows XP and Windows Server 2003 to generic names.

This change is not accurate. For IIS, the default account is NETWORK SERVICE on IIS 7 or earlier and is Application Pool Identity on IIS 7.5 or later. This is same for client and server skus. You can find more information [here](https://docs.microsoft.com/en-us/iis/manage/configuring-security/application-pool-identities).

Other than this, all other changes look good to me.

**Review by BillWagner — CHANGES_REQUESTED** [review]
Overall, this looks great.

Once you address the comments from @zhenlan we can merge this.

**Comment by mairaw** [comment]
@BillWagner @zhenlan sorry for the delay. I've addressed the comments. Thanks!

**Review by zhenlan — APPROVED** [review]
LGTM. Thanks!

**Review by BillWagner — APPROVED** [review]

**Comment by BillWagner** [comment]
Looks great. Once the build finishes, let's merge it.

**Pull request merged**